### PR TITLE
first rewrite

### PR DESCRIPTION
**Pull request by markedwardmurray**
@jmburges this is meant to follow https://github.com/learn-co-curriculum/reading-ios-creating-custom-class

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,66 @@
...
+## Objectives
+
+1. Create a custom class file that inherits from `NSObject`.
+2. Import another custom class.
+3. Declare a property on a custom class.
+4. Write methods for the custom class to perform on a property that it owns.
+
+## Instructions
+
+Fork and clone this lab.
```
Comment (jmburges) [inline README.md]: I would number these

Comment (jmburges) [inline README.md]: also you can drop the fork and clone, they should know that by now

File: iOShoppingCart/FISItem.m [hunk 2]
```diff
@@ -1,23 +1,21 @@
...
-#pragma mark - initializers
+- (instancetype)init {
+    self = [self initWithName:@""
+                 priceInCents:0];
+    return self;
+}
 
--(instancetype)initWithName:(NSString *)name andPrice:(NSNumber *)price
-{
+- (instancetype)initWithName:(NSString *)name
```
Comment (jmburges) [inline iOShoppingCart/FISItem.m]: Price probably should be a CGFloat. Things aren't always costing whole numbers.

File: iOShoppingCart/FISShoppingCart.h [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+@interface FISShoppingCart : NSObject
+
+@property (strong, nonatomic) NSMutableArray *items;
+
+- (NSUInteger)calculateTotalPriceInCents;
+- (void)addItem:(FISItem *)item;
+- (void)removeItem:(FISItem *)item;
+- (void)removeAllItemsLikeItem:(FISItem *)item;
+- (void)sortItemsByNameAsc;
```
Comment (jmburges) [inline iOShoppingCart/FISShoppingCart.h]: Add in a space to sorta section off the differen methods. The add/remove methods. The sort. the allItmes.

NitPicky

File: iOShoppingCartTests/FISItemSpec.m [hunk 4]
```diff
@@ -14,29 +8,31 @@
...
-    it(@"has a price",^{
-        expect(item).to.respondTo(@selector(price));
-        expect(item).to.respondTo(@selector(setPrice:));
-        expect(item.price).to.beKindOf([NSNumber class]);
-        expect(item.price).to.equal(@5);
+    describe(@"designated initializer", ^{
+        it(@"should set name and priceInCents to submitted argument values", ^{
+            expect(hotPockets.name).to.equal(@"Hot Pockets");
+            expect(hotPockets.priceInCents).to.equal(499);
+            expect(leanPockets.name).to.equal(@"Lean Pockets");
```
Comment (jmburges) [inline iOShoppingCartTests/FISItemSpec.m]: No need to test both! You're calling the same methods

**Comment by jmburges** [comment]
few nitpicks. but looks good. fix em and :shipit:

File: iOShoppingCart/FISItem.m (hunk 2 above)
Comment (markedwardmurray) [inline iOShoppingCart/FISItem.m]: NSUInteger priceInCents

**Pull request merged**